### PR TITLE
minikore interface proposal

### PR DESCRIPTION
**Pull request by daejunpark**
do not merge. just for a review.

### CODE REVIEW

File: src/main/scala/org/kframework/minikore/MiniKoreInterface.scala [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+  trait Pattern
+
+  trait Variable extends Pattern {
+    def name: String
+    def sort: String
+  }
+
+  trait Application extends Pattern {
+    def label: String
```
Comment (grosu) [inline src/main/scala/org/kframework/minikore/MiniKoreInterface.scala]: symbol not label anymore, right?

**Comment by daejunpark** [comment]
@grosu please take a look. we can talk more in person.

File: src/main/scala/org/kframework/minikore/MiniKoreInterface.scala (hunk 1 above)
Comment (daejunpark) [inline src/main/scala/org/kframework/minikore/MiniKoreInterface.scala]: Yes. I'll fix it in a separate PR.

**Comment by daejunpark** [comment]
@msaxena2 I just updated. Please take a look again.

**Comment by daejunpark** [comment]
@grosu @msaxena2 Here is 2nd proposal. Please take a look.

**Pull request closed without merging**